### PR TITLE
Add sample to get a specific version of the fixer

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Related to #1565.

cc @avindra

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -31,12 +31,16 @@ your system:
...
+    # With a specific version
+    $ wget http://get.sensiolabs.org/php-cs-fixer-v1.11.phar -O php-cs-fixer
 
 or with curl:
 
 .. code-block:: bash
 
     $ curl http://get.sensiolabs.org/php-cs-fixer.phar -o php-cs-fixer
+    # With a specific version
+    $ curl http://get.sensiolabs.org/php-cs-fixer-v1.11.phar.phar -o php-cs-fixer
```
Comment (gharlan) [inline README.rst]: `.phar.phar`

**Comment by keradus** [comment]
ping @Soullivaneuh

**Comment by Soullivaneuh** [comment]
@keradus fixed.

**Comment by keradus** [comment]
Thank you @Soullivaneuh.

**Pull request merged**